### PR TITLE
test: Run all consumers' message processors against examples

### PR DESCRIPTION
**Pull request by untitaker**
sentry-kafka-schemas is supposed to contain more and more examples over
time for all kafka topics. Run those examples against the relevant snuba
consumers.

### CODE REVIEW

File: snuba/processor.py [hunk 1]
```diff
@@ -101,6 +101,8 @@ class ReplacementType(str, Enum):
         ReplacementType.TOMBSTONE_EVENTS,
         ReplacementType.EXCLUDE_GROUPS,
         ReplacementType.REPLACE_GROUP,
+        ReplacementType.START_UNMERGE_HIERARCHICAL,
```
Comment (untitaker) [inline snuba/processor.py:104]: those message types definitely exist in Sentry, but the product logic is behind a featureflag nobody has. I suppose we would be crashing the consumer today if they aren't here.

File: tests/consumers/test_schemas.py [hunk 2]
```diff
@@ -1,107 +1,84 @@
...
+    ReplacerProcessor,
+)
+
+
+# Since these tests are pure, we can override the run_migrations fixture to
+# skip all test isolation. This reduces the test runtime from 10 seconds to 0.6
+# seconds
+@pytest.fixture
+def run_migrations():
+    pass
```
Comment (lynnagara) [inline tests/consumers/test_schemas.py:27]: Oh nice, we can probably speed up a whole lot of our test suites this way

File: snuba/processor.py [hunk 3]
```diff
@@ -101,6 +101,8 @@ class ReplacementType(str, Enum):
...
+        ReplacementType.END_UNMERGE_HIERARCHICAL,
```
Comment (lynnagara) [inline snuba/processor.py:105]: Good catch.. though this is literally just all the members of `ReplacementType` enum... Do we actually need to define this here at all?

**Review by lynnagara — APPROVED** [review]
Nice!

**Pull request merged**